<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TissueTypeEditorDialog</class>
 <widget class="QDialog" name="TissueTypeEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Наименование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblResetCounterType">
     <property name="text">
      <string>Период уникальности идентификатора</string>
     </property>
     <property name="buddy">
      <cstring>cmbCounterResetType</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cmbCounterResetType">
     <item>
      <property name="text">
       <string>День</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Неделя</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Месяц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Полгода</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Год</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Постоянно</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="chkCounterManualInput">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblCounterManualInput">
     <property name="text">
      <string>Ручной ввод идентификатора</string>
     </property>
     <property name="buddy">
      <cstring>chkCounterManualInput</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbSex">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ж</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblSex">
     <property name="text">
      <string>Пол</string>
     </property>
     <property name="buddy">
      <cstring>cmbSex</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblExternalIdLimit">
     <property name="text">
      <string>Ограничение длины идентификатора</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="edtExternalIdLimit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="edtCounterValue">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chkIdentCounter">
     <property name="text">
      <string>Счетчик идентификатора</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>cmbSex</tabstop>
  <tabstop>chkCounterManualInput</tabstop>
  <tabstop>cmbCounterResetType</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TissueTypeEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkIdentCounter</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtCounterValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TissueTypeEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkCounterManualInput</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkIdentCounter</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
